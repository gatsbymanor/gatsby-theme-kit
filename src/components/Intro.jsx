import React from 'react';
import picture from '../images/pic01.jpg'

export default function Intro(props) {
  return (
    <section id="intro" className="main">
      <div className="spotlight">
        <div className="content">
          <header className="major">
            <h2>Ipsum sed adipiscing</h2>
          </header>
          <p>Sed lorem ipsum dolor sit amet nullam consequat feugiat consequat magna
            adipiscing magna etiam amet veroeros. Lorem ipsum dolor tempus sit cursus.
            Tempus nisl et nullam lorem ipsum dolor sit amet aliquam.</p>
          <ul className="actions">
            <li><a href="generic.html" className="button">Learn More</a></li>
          </ul>
        </div>
        <span className="image"><img src={picture} alt="" /></span>
      </div>
    </section>
  )
}