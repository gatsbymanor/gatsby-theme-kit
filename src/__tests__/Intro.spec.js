import React from 'react';
import renderer from 'react-test-renderer';
import Intro from '../components/Intro';

describe('<Intro />', () => {
  const component = renderer.create(<Intro />).toJSON();

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});