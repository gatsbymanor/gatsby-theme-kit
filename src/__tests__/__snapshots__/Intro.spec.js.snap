// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Intro /> should render correctly 1`] = `
<section
  className="main"
  id="intro"
>
  <div
    className="spotlight"
  >
    <div
      className="content"
    >
      <header
        className="major"
      >
        <h2>
          Ipsum sed adipiscing
        </h2>
      </header>
      <p>
        Sed lorem ipsum dolor sit amet nullam consequat feugiat consequat magna adipiscing magna etiam amet veroeros. Lorem ipsum dolor tempus sit cursus. Tempus nisl et nullam lorem ipsum dolor sit amet aliquam.
      </p>
      <ul
        className="actions"
      >
        <li>
          <a
            className="button"
            href="generic.html"
          >
            Learn More
          </a>
        </li>
      </ul>
    </div>
    <span
      className="image"
    >
      <img
        alt=""
        src="images/pic01.jpg"
      />
    </span>
  </div>
</section>
`;
